frontend/src/charts/ChartSwitcher.svelte

```diff
@@ -72,4 +72,10 @@
   button:hover {
     color: var(--text-color-lighter);
   }
+
+  @media print {
+    button {
```

Comment:
mhh, let's maybe still show the currently selected one so that the chart has a label?

Reply:
Ah right!  I think the "line chart" doesn't provide much value, but the "Net Worth" absolutely would, what do you think of that approach, or would you prefer both?

Reply:
Realizing that's what you were asking for, and have updated